arch/arm/soc/atmel_sam/sam4s/Kconfig.soc

```diff
@@ -112,7 +105,15 @@ config SOC_ATMEL_SAM4S_PLLB_DIVB
 	Board config file can override this settings
 	for a particular board.
 
-	With default of MULB == 6, and DIVB == 1,
-	PLL is running at 7 times of main clock.
+	With default of MULB == 3, and DIVB == 1,
+	PLL is running at 4 times of main clock.
+
+config SOC_ATMEL_SAM4S_EFC0_FMR_FWS
```

Comment:
Yes, I believe that's the best move for now. Also, setting FWS to 5 is safe. Any other board with any other supply voltage will run, just not in an optimal way.